RichMessageKit/Views/ReceivedButtonsCell.swift

```diff
@@ -120,7 +120,15 @@ public class ReceivedButtonsCell: UITableViewCell {
             }
         }
 
-        nameLabel.text = model.message.displayName
+        // Check for Custom Bot Id then update name label
+        if let assigneeId = model.message.contactId,
+           assigneeId == RMCellConfiguration.customizedBotId,
+           !RMCellConfiguration.customBotName.isEmpty {
+            nameLabel.text = RMCellConfiguration.customBotName
```

Comment:
Can we move this logic to controller or VM? I don't think a subview should have this type of logic.
If ID matches, then pass the data to cell from there, otherwise not.

Reply:
done